### ISSUE TITLE
Results Reproduce

### ISSUE THREAD

**Issue by jakeiaCyber** [bjzhb666/GS-LoRA#12]
Hi, @bjzhb666 

Thank you for releasing this code.

I am currently facing some problems while reproducing the "Continual Forgetting Results for Face Recognition", and I hope you can help clarify them.

I am using NVIDIA GeForce RTX 3090, and cuda version is 11.4.

I follow the instructions in the README and execute `run_cl_forget.sh`. I review the `run_cl_forget.sh` and observe that the following code was executed as shown in the image.

<img width="1280" alt="Image" src="https://github.com/user-attachments/assets/f6326753-a928-414f-9b2f-a5f3d0960aa3" />

The following image shows the Task 0 section of my log. The results show a significant difference compared to TABLE 3 of the "Practical Continual Forgetting for Pre-trained Vision Models" paper.

<img width="609" alt="Image" src="https://github.com/user-attachments/assets/9d7c8cc8-567c-45a5-b63b-8b83a6675296" />

I hope you can provide me with some guidance on reproducing the GS-LoRA results in the "Continual Forgetting Results for Face Recognition."

I would greatly appreciate it. Once again, thank you for releasing this excellent work.

Here is my log.

[output.log](https://github.com/user-attachments/files/18905048/output.log)

**Comment by Mridul24003** [comment]
Hi, @bjzhb666.
Thank you so much for releasing the code. I was using your pre trained model checkpoint "Backbone_VIT_Epoch_1185_Batch_45020_Time_2024-09-26-03-26_checkpoint.pth " for face recognition task using your specified dataset Casia 100. 

After using this i tested the pre-trained model for the Face Transformer. To my surprise i am getting very low accuracy of around 48.553%. Can you provide me some guidance regarding the same. Are you ignoring some classes or is this the same accuracy that you are getting.

**Comment by bjzhb666** [comment]
> Hi, [@bjzhb666](https://github.com/bjzhb666). Thank you so much for releasing the code. I was using your pre trained model checkpoint "Backbone_VIT_Epoch_1185_Batch_45020_Time_2024-09-26-03-26_checkpoint.pth " for face recognition task using your specified dataset Casia 100.
> 
> After using this i tested the pre-trained model for the Face Transformer. To my surprise i am getting very low accuracy of around 48.553%. Can you provide me some guidance regarding the same. Are you ignoring some classes or is this the same accuracy that you are getting.

I guess you did something wrong, the checkpoint is correct.

**Comment by bjzhb666** [comment]
> Hi, [@bjzhb666](https://github.com/bjzhb666)
> 
> Thank you for releasing this code.
> 
> I am currently facing some problems while reproducing the "Continual Forgetting Results for Face Recognition", and I hope you can help clarify them.
> 
> I am using NVIDIA GeForce RTX 3090, and cuda version is 11.4.
> 
> I follow the instructions in the README and execute `run_cl_forget.sh`. I review the `run_cl_forget.sh` and observe that the following code was executed as shown in the image.
> 
> <img alt="Image" width="1280" src="https://private-user-images.githubusercontent.com/128603031/415599291-f6326753-a928-414f-9b2f-a5f3d0960aa3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIxOTYwNTUsIm5iZiI6MTc0MjE5NTc1NSwicGF0aCI6Ii8xMjg2MDMwMzEvNDE1NTk5MjkxLWY2MzI2NzUzLWE5MjgtNDE0Zi05YjJmLWE1ZjNkMDk2MGFhMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMxN1QwNzE1NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iOTA1YjI1YTc1YzM0ZGU0MDc4OGEzZTJhNmQ5Zjg5NGZkYjMyMGZmMTY5Yzg5NmY1ZDNkNDg3NDE4YjUyMWIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ga6moj09O3yzLwuWO9DTp-ZnPklZ4gcC99ENtB1yzLQ">
> The following image shows the Task 0 section of my log. The results show a significant difference compared to TABLE 3 of the "Practical Continual Forgetting for Pre-trained Vision Models" paper.
> 
> <img alt="Image" width="609" src="https://private-user-images.githubusercontent.com/128603031/415600078-9d7c8cc8-567c-45a5-b63b-8b83a6675296.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIxOTYwNTUsIm5iZiI6MTc0MjE5NTc1NSwicGF0aCI6Ii8xMjg2MDMwMzEvNDE1NjAwMDc4LTlkN2M4Y2M4LTU2N2MtNDVhNS1iNjNiLThiODNhNjY3NTI5Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMxN1QwNzE1NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YjQzNTcxYjMwMzlhYTk1YTVmMzlhYWI2OWRlNjhkYmJkZjhmNDdlM2ViZjBhNDQ4OWFhNjc4N2FkNzUyZTZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PbHXGOxUt8tfSfMWzyoYUNF-LXQYYf8wQ77dJrctqqE">
> I hope you can provide me with some guidance on reproducing the GS-LoRA results in the "Continual Forgetting Results for Face Recognition."
> 
> I would greatly appreciate it. Once again, thank you for releasing this excellent work.
> 
> Here is my log.
> 
> [output.log](https://github.com/user-attachments/files/18905048/output.log)

I will check the hyperparameters, thanks for pointing it out.
